packages/front/src/App.tsx

```diff
@@ -29,7 +30,7 @@ const useVerifyUserLogin = (
   useEffect(() => {
     if (user == null)
       verifyAnd(setUser).catch((_) => {
-        if (location.pathname != "/")
+        if (!(location.pathname == "/registrar" || location.pathname == "/"))
```

Comment:
this is fine, but if you want, here's another approach:
```suggestion
        if (!['/registrar', '/'].includes(location.pathname))
```